package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "type": "module",
   "scripts": {
-    "build": "tsc --build packages/* && tsc --build scripts",
+    "build": "tsc --build packages/* scripts",
```

Comment:
`--build` takes multiple source directories as arguments. The previous construction of this command meant that the `--watch` argument from the `watch` script (`"npm run build -- --watch"`) only applied to `tsc --build scripts`.